tokio/src/process/mod.rs

```diff
@@ -330,6 +330,11 @@ impl Command {
         &mut self.std
     }
 
+    /// Cheaply convert into a `std::process::Command`.
+    pub fn into_std(self) -> StdCommand {
```

Comment:
We may want a note to say that Tokio-specific options may not apply to the underlying std command?

Reply:
Is this new version okay?